owncloud/phoenix
# Feature/drawio integration

## Description
### First part:
Apps can now define if they want to be opened in their own browser tab including the capability to define the route to be used

### Second part:
an integration app for draw.io has been added ass ref implementation

## Motivation and Context
- elaborate on necessary changes to the files app to open up file editors/viewers in a new tab
- elaborate on capabilities to integrate a 3rd party web server

## How Has This Been Tested?
- create a new file test.drawio
- click on the file in the files list
- click on draw.io action 
- see file being opened in a second tab

### Note
- if the same file is opened twice the same tab will be reused
- a different file will be opened in a different tab

## Screenshots (if appropriate):
![Screenshot from 2019-09-27 11-15-47](https://user-images.githubusercontent.com/1005065/65757820-2fdd2e80-e118-11e9-994a-969b58cf78c7.png)
![Screenshot from 2019-09-27 11-15-36](https://user-images.githubusercontent.com/1005065/65757821-3075c500-e118-11e9-9afd-337abb4d69a7.png)

## Types of changes


## Checklist:



## Open tasks:
